### PR TITLE
move tiramisu to and fix up archive

### PR DESCRIPTION
**Pull request by SirGamers**
this includes the latest commits from main
[test site here](https://sirgamers.github.io/Guide-WiiU/)

### CODE REVIEW

**Comment by Lazr1026** [comment]
could you put Tiramisu under the cfw-choice? i dont see why it needs to be on the sidebar

**Comment by SirGamers** [comment]
it doesnt belong there cuz the cfw choice is for stuff not supported anymore, to quote the page "THE METHODS DESCRIBED IN THIS PAGE ARE NO LONGER SUPPORTED"

**Comment by SirGamers** [comment]
but tiramisu is still supported

**Comment by SirGamers** [comment]
[test site here](https://sirgamers.github.io/Guide-WiiU/)

**Comment by SirGamers** [comment]
i love how you can undo stuff using git

**Comment by SirGamers** [comment]
@Lazr1026 merge???

File: index.html [hunk 1]
```diff
@@ -72,8 +72,17 @@
                 '/privacy-policy'                                     : '/docs/privacy/privacy-policy',
 
                 // ARCHIVES
+                '/archive/tiramisu/sd-preparation'                    : '/docs/user-guide/archive/tiramisu/sd-preparation.md',
```
Comment (Lazr1026) [inline index.html]: is there a need for this twice? (see line 80)

**Comment by SirGamers** [comment]
fixed

File: docs/user-guide/archive/tiramisu/finalizing-setup.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+### Additional Homebrew Apps - Extracting Files to the SD Card
+
+1. Copy the contents of the newly downloaded `SaveMiiModWUTPort.zip` file to the root of your SD Card.
+1. Extract the contents of the newly downloaded `Bloopair_vX.X.X.zip` file.  
+   Copy the `30_bloopair.rpx` to the `wiiu/environments/tiramisu/modules/setup/` folder on the root of your SD Card.  
+   Copy the `wiiu` folder to the root of your SD Card.
+1. Copy the contents of the newly downloaded `wiiu-extracttosd.zip` file to the root of your SD Card.
+
+### Installing Aroma
```
Comment (Lazr1026) [inline docs/user-guide/archive/tiramisu/finalizing-setup.md]: Can probably just remove this section entirely

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

**Comment by Lazr1026** [comment]
sorry for late review, i was on vacation for thanksgiving

**Comment by SirGamers** [comment]
> sorry for late review, i was on vacation for thanksgiving

It’s fine

**Comment by SirGamers** [comment]
i removed it

**Pull request merged**